acceptance: TestDockerNodeJS failed

acceptance.TestDockerNodeJS [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15502803?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15502803?buildTab=artifacts#/) on master @ [80d0cced3780e77cffed6802f232af2860d62a81](https://github.com/cockroachdb/cockroach/commits/80d0cced3780e77cffed6802f232af2860d62a81):


```
=== RUN   TestDockerNodeJS
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/c08ff262c750a522d8c02a8762e12dc5/logTestDockerNodeJS2257584210
    test_log_scope.go:81: use -show-logs to present logs inline
    adapter_test.go:85: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/c08ff262c750a522d8c02a8762e12dc5/logTestDockerNodeJS2257584210
--- FAIL: TestDockerNodeJS (36.50s)
=== RUN   TestDockerNodeJS/Success
    util_docker.go:63: non-zero exit code: 1
    --- FAIL: TestDockerNodeJS/Success (22.49s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerNodeJS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-39195

acceptance.TestDockerNodeJS [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15513344?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15513344?buildTab=artifacts#/) on master @ [be9d35d273898834deb6082f921f445a93e60036](https://github.com/cockroachdb/cockroach/commits/be9d35d273898834deb6082f921f445a93e60036):


```
=== RUN   TestDockerNodeJS
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/c08ff262c750a522d8c02a8762e12dc5/logTestDockerNodeJS2404291799
    test_log_scope.go:81: use -show-logs to present logs inline
    adapter_test.go:85: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/c08ff262c750a522d8c02a8762e12dc5/logTestDockerNodeJS2404291799
--- FAIL: TestDockerNodeJS (39.17s)
=== RUN   TestDockerNodeJS/Success
    util_docker.go:63: non-zero exit code: 1
    --- FAIL: TestDockerNodeJS/Success (22.14s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerNodeJS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestDockerNodeJS [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15514166?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/15514166?buildTab=artifacts#/) on master @ [be9d35d273898834deb6082f921f445a93e60036](https://github.com/cockroachdb/cockroach/commits/be9d35d273898834deb6082f921f445a93e60036):


```
=== RUN   TestDockerNodeJS
    test_log_scope.go:170: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/c08ff262c750a522d8c02a8762e12dc5/logTestDockerNodeJS2285451812
    test_log_scope.go:81: use -show-logs to present logs inline
    adapter_test.go:85: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/c08ff262c750a522d8c02a8762e12dc5/logTestDockerNodeJS2285451812
--- FAIL: TestDockerNodeJS (66.13s)
=== RUN   TestDockerNodeJS/Success
    util_docker.go:63: non-zero exit code: 1
    --- FAIL: TestDockerNodeJS/Success (47.64s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerNodeJS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/acceptance/adapter_test.go
+++ b/pkg/acceptance/adapter_test.go
@@ -74,7 +74,8 @@ func TestDockerNodeJS(t *testing.T) {
 	export SHOULD_FAIL=%v
 	# Get access to globally installed node modules.
 	export NODE_PATH=$NODE_PATH:/usr/lib/node
-	/usr/lib/node/.bin/mocha .
+	# Have a 10 second timeout on promises, in case the server is slow.
+	/usr/lib/node/.bin/mocha -t 10000 . 
 	`
 
 	ctx := context.Background()

